### ISSUE TITLE
Using Proj4 loads a library which is incompatible with Git

### ISSUE THREAD

**Issue by asinghvi17** [JuliaGeo/Proj4.jl#38]
This causes:
```julia
julia> using Proj4
[ Info: Precompiling Proj4 [9a7e659c-8ee8-5706-894e-f68f43bc57ea]

julia> run(`git rev-parse HEAD`)
dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib
  Referenced from: /usr/local/bin/git
  Reason: Incompatible library version: git requires version 10.0.0 or later, but libpcre2-8.0.dylib provides version 8.0.0
```

**Comment by visr** [comment]
Hmm. Does this also occur on the `artifact` branch of #33? Would be another reason to finish that up.

**Comment by asinghvi17** [comment]
Using the artifact branch fixes it, so this should be fixed by #33.  Is there a particular reason that hasn't been merged yet, BTW?

**Comment by visr** [comment]
The last open todo there is "Integrate the artifacts with the rest of the package and tests it".

I haven't really used the proj-datumgrid files in practice. I'm not sure how best to integrate it. Perhaps I should just leave it. People can always still use the artifacts themselves.

**Comment by visr** [comment]
Fixed by #33, which is released as Proj4.jl v0.6.0
